SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -569,6 +481,29 @@ void GenerateSubobjectSchema(FComponentIdGenerator& IdGenerator, UClass* Class,
 	SubobjectClassPathToSchema.Add(Class->GetPathName(), SubobjectSchemaData);
 }
 
+EReplicatedPropertyGroup SchemaComponentTypeToPropertyGroup(ESchemaComponentType SchemaType)
```

Comment:
nit - would rather this was after PropertyGroupToSchemaComponentType

Reply:
Ah, yes, I also wanted it like that, unfortunately PropertyGroupToSchemaComponentType is in an anonymous namespace, and I have to export this one :(
Maybe something else could be done?

Reply:
er.. you could move this to namespace, and expose a function that just calls into it?
not fussed though

Reply:
perhaps we should expose PropertyGroupToSchemaComponentType ?